### ISSUE TITLE
Thoughts on new passphrase feature

### ISSUE THREAD

**Issue by kyuupichan** [spesmilo/electrum#1917]
Not a bug per se, something to think about.

I'm not sure, but perhaps an empty passphrase should be saved encrypted in the wallet file (the keystore seems to have nothing if no passphrase is set).  There may be a plausible deniability benefit in an adversary not knowing if a passphrase was used.

Having said that, it may also be useful (for confused user support) to know whether a passphrase was used or not...

**Comment by AbdussamadA** [comment]
relevant:

https://bitcointalk.org/index.php?topic=812894.msg10900704#msg10900704

**Comment by ecdsa** [comment]
@AbdussamadA I tend to agree with gmaxwell about deniability. anything human generated is weak.

I added the passphrase feature so that passphrases can be used as enhanced 'account numbers', allowing to use several accounts with the same seed. See my point 3 in this email: https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2016-August/013061.html

**Comment by AbdussamadA** [comment]
@ecdsa Thanks for pointing that out. I hadn't thought of using it like that. Also thanks for your help with the other issue.
